phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2853,12 +2687,13 @@ MetaDataMutationResult updateMutation(PTable table, byte[][] rowKeyMetaData,
                     return result;
                 } else {
                     table = buildTable(key, cacheKey, region, HConstants.LATEST_TIMESTAMP, clientVersion, false, false, null);
-                    return new MetaDataMutationResult(MutationCode.TABLE_ALREADY_EXISTS, currentTime, table);
```

Comment:
Is there a reason the MutationCode returned needs to change?

Reply:
You can add a new COLUMN_MODIFIED MutationCode, but no need to change the existing MutationCode returned for drop column etc.